boat_nav/src/layline_action.py

```diff
@@ -49,43 +50,19 @@ def target_callback(self, new_target):
 	def gps_callback(self, gps):
 		self.boat_speed = gps.speed * 0.514444 # Knots to m/s
 
-	def is_within_bounds(self, val, boundA, boundB):
-		if boundA < boundB:
-			val -= boundA
-			boundB -= boundA
-			boundA = 0
-		else:
-			val -= boundB
-			boundA -= boundB
-			boundB = 0
-		if val < 0:
-			val += 360
-		val = val % 360
-		return (boundA <= val and val <= boundB) or (boundB <= val and val <= boundA)
-
-	def gtAngle(self, angle1, angle2):
-		comp_angle = (angle2 + 180) % 360
-		if angle2 >= 180:
-			return not self.is_within_bounds(angle1, angle2, comp_angle)
-		else:
-			return self.is_within_bounds(angle1, angle2, comp_angle)
-	
-	def ltAngle(self, angle1, angle2):
-		comp_angle = (angle2 + 180) % 360
-		if angle2 >= 180:
-			return self.is_within_bounds(angle1, angle2, comp_angle)
-		else:
-			return not self.is_within_bounds(angle1, angle2, comp_angle)
-
 	def anemometer_callback(self, new_heading):
 		self.ane_reading = new_heading.data
-		self.apparent_wind_heading = (self.ane_reading + self.compass) % 360
-		self.wind_coming = (self.apparent_wind_heading + 180) % 360
+		self.update_apparent_wind()
+
 
 	def compass_callback(self, compass):
 		self.compass = compass.data
-		self.apparent_wind_heading = (self.ane_reading + self.compass) % 360
-		self.wind_coming = (self.apparent_wind_heading + 180) % 360
+		self.update_apparent_wind()
+
+	def update_apparent_wind(self):
+		self.apparent_wind_heading = angles.normalize(self.ane_reading + self.compass)
+		self.wind_coming =  angles.normalize(self.apparent_wind_heading + 180)
```

Comment:
Feel like normalize(angle + 180) should have its own utility like `flip` or `reflect` or something, seems like we add 180 without much explanation in a lot of places